lib/SILOptimizer/Analysis/EscapeAnalysis.cpp

```diff
@@ -457,23 +457,17 @@ void EscapeAnalysis::ConnectionGraph::initializePointsTo(CGNode *initialNode,
                                                          bool createEdge) {
   // Track nodes that require pointsTo edges.
   llvm::SmallVector<CGNode *, 4> pointsToEdges;
```

Comment:
The name "pointsToEdges" is a bit misleading. This vector contains nodes either nodes which have the pointsTo set, even if it's not an edge, and nodes, which need to to get a pointsToEdge.
Though, I don't have a bette idea for a name.

Reply:
Fixed `pointsToEdgeNodes`